test/resolvers/role.js

```diff
@@ -161,5 +168,31 @@ describe('Role resolver', () => {
           assert.equal(establishment.updatedAt, updatedAt);
         });
     });
+
+    it('removing a nacwo also dissociates the user from any related places', () => {
+      const opts = {
+        action: 'delete',
+        id: NACWO_ROLE_ID,
+        data: {
+          establishmentId: ESTABLISHMENT_ID,
+          profileId: PROFILE_ID
+        }
+      };
+      return Promise.resolve()
+        .then(() => this.models.Place.query().insert({
+          site: 'Site 1',
+          area: 'Area 1',
+          name: 'Some Place',
+          suitability: ['DOG'],
+          holding: ['STH', 'LTH'],
+          establishmentId: ESTABLISHMENT_ID,
+          nacwoId: NACWO_ROLE_ID
+        }))
+        .then(() => this.role(opts))
+        .then(() => this.models.Place.query().where({ nacwoId: NACWO_ROLE_ID }))
+        .then(places => {
+          assert.equal(places.length, 0, 'there should be no places with that nacwo role id');
+        });
+    });
```

Comment:
Would be good to also have a test that it does _not_ disassociate from other establishments.